packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -44,7 +45,7 @@ const focusMap = new Map<List["el"], number>();
 export class ListItem extends LitElement implements InteractiveComponent, SortableComponentItem {
   //#region Static Members
 
-  static override styles = styles;
+  static override styles = [listItemStyles, sortableStyles];
```

Comment:
We can address this in the refactor issue, but we could make it a convention to use `styles` for the component styles and only name addition/extension styles.